apps/native/app/src/screens/document-detail/document-reply.tsx

```diff
@@ -0,0 +1,241 @@
+import React, { useEffect, useState } from 'react'
+import { useIntl } from 'react-intl'
+
+import { NavigationFunctionComponent } from 'react-native-navigation'
+
+import styled from 'styled-components/native'
+import { LoadingIcon } from '../../components/nav-loading-spinner/loading-icon'
+import { Pressable } from '../../components/pressable/pressable'
+import { useUser } from '../../contexts/user-provider'
+import { useDocumentReplyMutation } from '../../graphql/types/schema'
+import { createNavigationOptionHooks } from '../../hooks/create-navigation-option-hooks'
+import { useNavigationModal } from '../../hooks/use-navigation-modal'
+import { useAuthStore } from '../../stores/auth-store'
+import {
+  Button,
+  Container,
+  Icon,
+  NavigationBarSheet,
+  Problem,
+  Spacing,
+  TextField,
+  Typography,
+} from '../../ui'
+import { ComponentRegistry } from '../../utils/component-registry'
+import { isAndroid } from '../../utils/devices'
+
+const Host = styled.SafeAreaView`
+  flex: 1;
+`
+
+const Row = styled(Container)<{
+  hasBottomBorder?: boolean
+  paddingVertical?: Spacing
+}>(({ hasBottomBorder = true, paddingVertical = 1, theme }) => ({
+  flexDirection: 'row',
+  paddingTop: theme.spacing[paddingVertical],
+  paddingBottom: theme.spacing[paddingVertical],
+  ...(hasBottomBorder && {
+    borderBottomWidth: theme.border.width.standard,
```

Comment:
Should use [StyleSheet.hairlineWidth](https://reactnative.dev/docs/stylesheet#hairlinewidth) for most UI borders like these. Eg design uses 0.5 width border for a retina screen.